AsyncDisplayKit/ASCellNode.h

```diff
@@ -167,6 +161,19 @@ typedef NS_ENUM(NSUInteger, ASCellNodeVisibilityEvent) {
  */
 - (void)cellNodeVisibilityEvent:(ASCellNodeVisibilityEvent)event inScrollView:(nullable UIScrollView *)scrollView withCellFrame:(CGRect)cellFrame;
 
+#pragma mark - UITableViewCell specific passthrough properties
+
+/*
+ * ASTableView uses these properties when configuring UITableViewCells that host ASCellNodes.
```

Comment:
Let's add to the description: "These must be set before the cell comes onscreen to take effect."